helm_chart/HyperPodHelmChart/values.yaml

```diff
@@ -138,11 +138,57 @@ nvidia-device-plugin:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms: 
         - matchExpressions:
-          # nvidia plugin needs at least one node selector. Below label exists for all hyperpod nodes
```

Comment:
Original `kubernetes.io/os` selector was removed because matchExpressions are OR-ed and so keeping it would not change status quo.

For reference, default `nodeAffinity` config of nvd-device-plugin can be viewed at https://github.com/NVIDIA/k8s-device-plugin/blob/4eb8001984b148a339114ba86e9e20abdcbc92d3/deployments/helm/nvidia-device-plugin/values.yaml#L65C5-L84 . All those `nodeAffinity` criterias are for some special cases that don't apply to hyperpod so overriding it here should be ok.